docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,171 @@
+
+[Standard](https://standardjs.com/) is the JavaScript linter.
+Please ensure any new code is Standard-complaint by running the linter manually. It is installed as a dev dependency.
+There are also [text editor plugins](https://standardjs.com/index.html#are-there-text-editor-plugins) available.
+
+```
+npm install
+standard assets/src
+```
+
+#### Backend
+
+The MyLA backend is build using the [Django](https://www.djangoproject.com/) framework and
+uses [MySQL](https://www.mysql.com/) for a database platform.
+
+MyLA can only be run in production as an LTI tool. Previous versions of MyLA supported SAML,
+but this has been removed with a preference on launching via LTI Advantage.
+
+Included in the source code is a cron job ([`dashboard/cron.py`](../dashboard/cron.py)) that is used for
+populating the Canvas context and event data in the MySQL database.
+Each institution may need to configure various settings differently based on their infrastructure.
+More info on various institutions' infrastructure set-up is
+[here](https://github.com/tl-its-umich-edu/my-learning-analytics/wiki/Deploy:-Institution-Architectures).
+
+### Create an issue
+
+Before sending a pull request, please create an
+[issue](https://github.com/tl-its-umich-edu/my-learning-analytics/issues/new)
+describing either a problem (i.e. bug) in MyLA or a feature (i.e. enhancement) to be contributed.
+We will do our best to review the issue in a timely manner to discuss before starting work to address the issue.
+
+### Creating pull requests
+
+Once the issue has been discussed with one of the project maintainers, please follow these steps for creating a
+[pull request](https://github.com/tl-its-umich-edu/my-learning-analytics/pulls).
+
+1. Fork [this project](https://github.com/tl-its-umich-edu/my-learning-analytics) on GitHub by pressing the
+`Fork` button in the top-right hand corner of the repository's home page.
+
+1. Clone the forked repository to the local machine.
+    ```
+    git clone https://github.com/{github-username}/my-learning-analytics.git
+    ```
+
+1. Add an `upstream` [remote](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) to the
+main project repository.
+    ```
+    git remote add upstream https://github.com/tl-its-umich-edu/my-learning-analytics.git
+    ```
+
+1. Create local branches to keep contributions organized (generally one branch for each issue).
+    ```
+    git checkout -b issue-xxx-fix-a-bug
+    ```
+
+1. Make changes and push to your fork (`origin` remote).
+    ```
+    git add .
+    git commit -m 'Make some changes'
+    git push origin issue-xxx-fix-a-bug
+    ```
+
+1. Once changes have been pushed to the branch on your fork,
+GitHub will show a button to create a pull request from the forked repository to the main project repository.
+Click this and provide details about the changes in the description,
+making sure to mention (using #) the issue you want to resolve.
```

Comment:
Can make a recommendation here to include test plan as best practice.

Reply:
Okay, I can add that.

Reply:
See https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1398/commits/74704d4d1970042fc49301591cf502e00018441a